### DISCUSSION TITLE
setting to turn off animated gifs

### DISCUSSION THREAD

**Discussion by lottepeisch** [civitai/civitai#271] [Ideas]
Few days back, I couldn't figure out why the site kept spiking my Vivaldi CPU and GPU process, which seemed weird on a 2019 tricked out Macbook Pro. I couldn't figure it out, but I finally did today. Whenever an animated profile pics or model pics get loaded or pre-loaded, my GPU spikes like mad. As soon as they fall out, GPU goes back to normal. Very reproducible. 

They're also extremely distracting, and there's only three that I can count right now and they're already giving me a headache. I suspect it won't be long before we're partying like it's 1999 again, if you know what I mean... 

So... gimme a way to turn it off or stop the cycle after one run-through so I can opt out of the madness?

**Comment by Judas87Swiff** [comment]
It would also be easier to avoid "Buzz" farming by using Gifs unfairly to redirect to a model that will at this point never have Gif making functions directly through Civit.Ai's In-Browser Image Generator.
